cypress/integration/commandPalette.js

```diff
@@ -1,17 +1,14 @@
 describe('Command Palette', () => {
-    beforeEach(() => {
-        cy.visit('/events')
-    })
-
     it('Shows on toggle button click', () => {
         cy.get('[data-attr=command-palette-toggle]').click()
         cy.get('[data-attr=command-palette-input]').should('exist')
     })
 
     it('Shows on Ctrl + K press', () => {
-        cy.get('body').type('{ctrl}C')
+        cy.get('body').type('{ctrl}k')
         cy.get('[data-attr=command-palette-input]').should('exist')
-        cy.get('body').type('{cmd}C')
```

Comment:
Wow, this was dumb.

Reply:
:sweat_smile: took me like 5 minutes to figure out what the "bug" is.